src/store/modules/layers.store.js

```diff
@@ -181,9 +190,9 @@ const actions = {
                 // the KML layers, in this case we take the old active configuration as fallback.
                 // For KML layers the configuration doesn't requires translation as they only have
                 // the layer name to be translated and it is done dynamically using a getter name()
-                commit('addLayerWithConfig', layer.clone())
+                commit('addLayer', { layer: layer.clone() })
```

Comment:
done

Reply:
I don't see the fix. See screenshot attached below. I have changed the language to french but the drawing layer name is still in german "Zeichnung". But this can also be fixed in another PR.
![image](https://user-images.githubusercontent.com/55620677/214100564-67ca25f8-60be-430f-8f9f-5d2f02b69ee2.png)

Reply:
I changed the comment, I don't think this name should change by design, it is translated during the creation of the drawing using the current language at that time and then it is part of the URL. On the old viewer it is not re-translated if you don't have the adminId, only if you have adminId it is re-translated in case of language changes.
IMHO we should leave it for the moment as is, and we might in future improve it by allowing the user to define its own title and then it won't be anymore translated.